src/RTGS.DotNetSDK/RTGS.DotNetSDK.IntegrationTests/Subscriber/TestData/SubscriberActions.cs

```diff
@@ -9,8 +9,15 @@ namespace RTGS.DotNetSDK.IntegrationTests.Subscriber.TestData;
 
 public static class SubscriberActions
 {
+	private static Dictionary<string, string> _defaultSigningHeaders = new Dictionary<string, string>
+	{
+		{ "public-did-signature", "public-did-signature" },
+		{ "pairwise-did-signature", "pairwise-did-signature" },
+		{ "alias", "alias" }
+	};
+
 	public static readonly SubscriberAction<FIToFICustomerCreditTransferV10> PayawayFundsV1 =
-		new(new AllTestHandlers(), handlers => handlers.OfType<TestPayawayFundsV1Handler>().Single(), "PayawayFunds", ValidMessages.PayawayFunds);
+		new(new AllTestHandlers(), handlers => handlers.OfType<TestPayawayFundsV1Handler>().Single(), "PayawayFunds", ValidMessages.PayawayFunds, _defaultSigningHeaders);
```

Comment:
If you implement the above suggestion...

```suggestion
		new(new AllTestHandlers(), handlers => handlers.OfType<TestPayawayFundsV1Handler>().Single(), "PayawayFunds", ValidMessages.PayawayFunds, DefaultSigningHeaders);
```